README.md

```diff
@@ -54,6 +54,12 @@ Change summary
 |--------------|-----------------------------------------|
 | Gravity star | Pull time reduced by -33% (2.0 -> 1.33) |
 
+### Pilot classes
```

Comment:
```suggestion
### Tacticals
```

Might be better?

Also I would move this above the `Ordnances` section because tacticals are more relevant

Reply:
now fixed